README.md

```
@@ -311,6 +311,51 @@ The `shutdown` method performs the following cleanup:
+// Perform some cache operations
+await RunCache.set({ key: "user-1", value: "Alice" });
+await RunCache.get("user-1");
+
+// Disable verbose logging when done debugging
+RunCache.configure({ verbose: false });
+```
+
+### Log Levels
+
+Logs are output at different levels depending on their importance:
+- `info`: General information about cache operations
```

markdownlint-cli2 (0.17.2)

Lists should be surrounded by blank lines
null

(MD032, blanks-around-lists)